nixos/modules/services/hardware/bitbox-bridge.nix

```diff
@@ -0,0 +1,77 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.services.bitbox-bridge;
+in
+{
+  options = {
+    services.bitbox-bridge = {
+      enable = lib.mkOption {
+        type = lib.types.bool;
+        default = false;
+        description = ''
+          Enable Bitbox bridge daemon, for use with Bitbox hardware wallets.
+        '';
+      };
+
+      package = lib.mkPackageOption pkgs "bitbox-bridge" { };
+
+      port = lib.mkOption {
+        type = lib.types.port;
+        default = 8178;
+        description = ''
+          Listening port for the bitbox-bridge.
+        '';
+      };
+
+      runOnMount = lib.mkOption {
+        type = lib.types.bool;
+        default = false;
+        description = ''
+          Enable running bitbox-bridge.service only when hardware wallet is plugged, also registers the systemd device unit. When false, bitbox-bridge service runs all the time.
+        '';
+      };
```

Comment:
```suggestion
      runOnMount = lib.mkEnableOption ''
        running bitbox-bridge.service only when hardware wallet is plugged, also registers the systemd device unit. 
        When false, bitbox-bridge service runs all the time
      '';
```

Reply:
I've changed the logic to make config.bitbox-bridge.runOnMount = true by default, seems like a better default to conserve power/battery. Let me know if you still think we should change it ;)